### PR TITLE
Users/ endpoint

### PR DESCRIPTION
**Pull request by tsa96**
Small PR, just finishing up the recent work on users/, this pretty much finalises it. There's a few loose ends that will get tied up with the user/ endpoint work like Discord, Twitch and Twitter auth.

All the old tests were ported, and I wrote a bunch of new ones. Feels good to finally have an endpoint with complete tests, that's passing everything!

### CODE REVIEW

**Comment by tsa96** [comment]
Oh, I just noticed Swagger is failed to build now. Which is weird since I don't think I've touched anything besides Jest tests since I last ran in development - it was running fine after the package upgrades. Whatever, I'll fix it up tomororw.

**Comment by tsa96** [comment]
Resolved the Swagger circular dependency issues, with the minimal amount of lazy resolving. I'll work on the Swagger CLI plugin w/ testingstuff in the next week or so like I discussed in Discord. That'll ensure these are caught be Jest in the future.

**Comment by tsa96** [comment]
Users/ getall maprank now in, and CD stuff should be resolved.

**Comment by TheNerdMan** [comment]
Apologies for not looking at this sooner, will be taking a look at the branch now the CD stuff is fixed

**Review by TheNerdMan — APPROVED** [review]
WillFerrellEvilBossGIF

File: server/src/modules/users/users.service.ts [hunk 1]
```diff
@@ -46,8 +46,8 @@ export class UsersService {
         if (search) where.alias = { startsWith: search };
 
         const include: Prisma.UserInclude = {
-            profile: expand?.includes('profile'),
-            userStats: expand?.includes('userStats')
+            profile: !!expand?.includes('profile'),
```
[the remark this answers is not in the document]
Comment (tsa96) [reply server/src/modules/users/users.service.ts:50]: This is done on my current branch btw

**Pull request merged**